docs/src/main/paradox/server-side/low-level-api.md

```diff
@@ -158,8 +158,8 @@ HTTP pipelining is generally discouraged (and [disabled by most browsers](https:
 is nevertheless fully supported in Akka HTTP. The limit is applied on two levels. First, there's the
 `akka.http.server.pipelining-limit` config setting which prevents that more than the given number of outstanding requests
 is ever given to the user-supplied handler-flow. On the other hand, the handler flow itself can apply any kind of throttling
-itself. If you use one of the `Http.bindAndHandleSync` or `Http.bindAndHandleAsync`
-entry-points, you can specify the `parallelism` argument (default = 1, i.e. pipelining disabled) to control the
+itself. If you use the `Http.bindAndHandleAsync`
+entry-point, you can specify the `parallelism` argument (default = 1, i.e. pipelining disabled) to control the
```

Comment:
`bla = 1` syntax reads weird in documentation, this is prose, not code :) Will adjust and merge, thanks!